src/tesk_core/job.py

```diff
@@ -34,9 +39,15 @@ def run_to_completion(self, poll_interval, check_cancelled, pod_timeout):
                 raise ApiException(ex.status, ex.reason)
         is_all_pods_running = False
         status, is_all_pods_running = self.get_status(is_all_pods_running)
+        # notify the callback receiver that the job is running
+        if self.callback and status == 'Running':
```

Comment:
Again, better to compare explicity against `None`, i.e., if `self.callback is not None [...]`. Same below.